Changelog.md

```diff
@@ -17,6 +17,7 @@ Bugfixes:
  * IR Generator: Fix IR syntax error when copying storage arrays of structs containing functions.
  * Natspec: Fix ICE when overriding a struct getter with a Natspec-documented return value and the name in the struct is different.
  * TypeChecker: Fix ICE when a constant variable declaration forward references a struct.
+ * Fix ICE when an external function has additional slots (Gas, Value etc).
```

Comment:
Maybe something like this :-)?
```suggestion
 * Code Generator: Fix ICE when accessing the address member of external functions types occupying more than two stack slots.
```

Reply:
If we also do this for ``.selector`` here, we can change ``the address member`` to ``members`` :-).